docs/source/platform/schema-registry.md

```diff
@@ -68,7 +66,7 @@ The only change in this push is the addition of the `--tag` flag on the end of t
 
 <h3 id="sending-tagged-metrics">Running a tagged schema</h3>
 
-To get the most out of using tagged schemas, teams can send metrics to [Engine](https://engine.apollographql.com) associated with this tag. This enables a single service to be tracked in production, staging, and any other environment a schema is being run. To track metrics with a schema, make sure the latest Apollo Server is installed and turn on tagging in one of two ways:
+To get the most out of tagged schemas, teams can send metrics to [Engine](https://engine.apollographql.com) with this tag. This enables a single service to be tracked in production, staging, and any other environment running a schema. To associate metrics with a schema, make sure the latest Apollo Server is installed and turn on tagging in one of two ways:
```

Comment:
I don't understand when we want to say "Engine" vs. "The Apollo UI," and this sentence is a great example of why it's unclear. Let's take the renaming out of this PR until we understand when/why we would do that.